app/views/layouts/application.html.erb

```diff
@@ -44,20 +44,8 @@
      </li>
    </div>
     <% end %>
-  <ul>
-      </li>
-</ul>
-      <div class="footer">
-      <div class="boxes">
-        <div class="box box1">
-          <div>
-            <p> </p>
-            <p> University of Cincinnati Libraries <br>
-              PO Box 210033 <br>
-              Cincinnati, Ohio <br>
-              45221-0033 <br>
-                <span class="dark"><a href="https://libraries.uc.edu/about/contact.html">Contact Us</a> </span>| <a href="https://libapps.libraries.uc.edu/main/contact/staff_directory.php">Staff Directory</a></p>
-        
+    <%= render partial: 'layouts/footer' %>
+    <div class="signin"
```

Comment:
Missing a closing bracket.  Might cause issues.

Reply:
where is the missing > ?

Reply:
![Screenshot 2024-04-01 at 1 07 42 PM](https://github.com/uclibs/staff-directory-23/assets/57018862/7f166e16-9290-469e-8057-55239fe98339)

Reply:
the missing > is fixed !